modules/_labs/core/react/stories/stories_direction.tsx

```diff
@@ -0,0 +1,56 @@
+/// <reference path="../../../../../typings.d.ts" />
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import withReadme from 'storybook-readme/with-readme';
+
+import {IconButton, IconButtonProps} from '../../../../button/react/index';
```

Comment:
Can you change this to `@workday/canvas-kit-react-button` and add it as a devDependency?